### PR TITLE
Added UserController tests

### PR DESCRIPTION
**Pull request by raymondwu1**

### CODE REVIEW

**Comment by raymondwu1** [comment]
addresses issue #13

File: src/test/java/com/UserControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    @MockBean
+    private UserService userService;
+
+    @MockBean
+    private UserValidator userValidator;
+
+    @Test
+    public void homeShouldReturnHome() throws Exception{
+        this.mvc.perform(get("/home")).andDo(print())
```
Comment (DanHoganRed) [inline src/test/java/com/UserControllerTest.java]: Why print?

Comment (raymondwu1) [reply src/test/java/com/UserControllerTest.java]: Yea that is unnecessary, will fix.

**Comment by DanHoganRed** [comment]
Cool, works locally. Ship it!

**Pull request merged**